yt_dlp/extractor/elonet.py

```diff
@@ -23,8 +22,9 @@ class ElonetIE(InfoExtractor):
             'ext': 'mp4',
             'title': 'Valkoinen peura',
             'description': 'Valkoinen peura (1952) on Erik Blombergin ohjaama ja yhdessä Mirjami Kuosmasen kanssa käsikirjoittama tarunomainen kertomus valkoisen peuran hahmossa lii...',
-            'thumbnail': 'https://elonet.finna.fi/Cover/Show?id=kavi.elonet_elokuva_107867&index=0&size=large',
+            'thumbnail': 'https://elonet.finna.fi/Cover/Show?id=kavi.elonet_elokuva_107867&index=0&size=large&source=Solr',
         },
+        'skip': 'Site no longer provides m3u8 streams',
```

Comment:
so? The test still works, no? Just remove the comment above

PS: The md5 test will likely fail. You can remove it and add `skip_download`